### ISSUE TITLE
Improve drag and drop events

### ISSUE THREAD

**Issue by wwwouaiebe** [wwwouaiebe/leaflet.TravelNotes#8]
Drag and drop can be ennoying because interaction with the input event...

**Comment by wwwouaiebe** [comment]
Seems to be a FF and a MS Edge bug. The caret cannot be moved with the mouse when draggable is true on th input element or on a parent element.
OK on Vivaldi...
Try this:
```
<!DOCTYPE html>
<html>
	<head>
		<meta charset='UTF-8' />
	</head>
	<body>
		<div id='idDiv'>
			<input id = 'idInput' type='text' Value='This is an input element' />
		</div>
		<script>
			document.getElementById ( 'idInput' ).draggable = true;
		</script>
	</body>
</html>
```

**Comment by wwwouaiebe** [comment]
Deactivating draggable on focus and reactivating on blur...

**Comment by wwwouaiebe** [comment]
Fixed in v1.0.0-RC1
